README.md

```diff
@@ -8,7 +8,62 @@ Discord: [Ante Finance](https://discord.gg/yaJthzNdNG)
 
 ---
 
-_The following has been provided by [@waynebruce0x](https://github.com/waynebruce0x):_
+## If you want to contribute to the community repository see the instructions below
+
+### Installation
+
+Install Node and NPM ([Instructions](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm))
+
+```
+Make sure to use Node version 14 or higher and NPM version 6 or higher
+```
+
+Clone the ante-v0-core repository
```

Comment:
Shouldn't this be
"Fork the ante-community-tests repository" 
"Clone your own fork with"
```
git clone git@github.com:[YOUR_GITHUB_USERNAME]/ante-community-tests.git
```